src/DynamoCoreWpf/Interfaces/IVisualizationManager.cs

```diff
@@ -68,6 +68,20 @@ public interface IVisualizationManager : IDisposable
         IRenderPackage CreateRenderPackageFromGraphicItem(IGraphicItem graphicItem);
 
         /// <summary>
+        /// An event triggered on the Selection of model.
+        /// </summary>
+        event Action<IEnumerable> RenderSelection;
```

Comment:
@ramramps Can we rename this to `VisualizationManager.SelectionHandled`, the fact that a rendering occurs is a downstream concern that should not affect the name.